src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.iOS.cs

```diff
@@ -564,12 +564,6 @@ void LoadRecognizers()
 			if (ElementGestureRecognizers == null)
 				return;
 
-			if (_shouldReceiveTouch == null)
-			{
-				// Cache this so we don't create a new UITouchEventArgs instance for every recognizer
-				_shouldReceiveTouch = ShouldReceiveTouch;
-			}
```

Comment:
C# 11 and higher does this for you now: https://github.com/dotnet/roslyn/issues/5835